vocabularies/borehole-status-event.ttl

```diff
@@ -123,3 +124,28 @@ bhse:manual-update a skos:Concept ;
     skos:inScheme <http://linked.data.gov.au/def/borehole-status-event> ;
     skos:prefLabel "Manual Update or Creation"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/borehole-status-event> .
+
+#COLLECTIONS
+
+bhse:pggd-01 a skos:Collection ;
+    skos:definition "The borehole events that may be reported via a PGGD-02 form to change the status of a borehole."@en ;
+    skos:member bhse:drilling-of-borehole-on-notice ;
+    skos:prefLabel "PGGD-01"@en .
+
+bhse:pggd-02 a skos:Collection ;
+    skos:definition "The borehole events that may be reported via a PGGD-02 form to change the status of a borehole."@en ;
+    skos:member bhse:completion,
+        bhse:alteration,
+        bhse:abandonment ;
+    skos:prefLabel "PGGD-02"@en .
+
+bhse:pggd-03 a skos:Collection ;
+    skos:definition "The borehole events that may be reported via a PGGD-02 form to change the status of a borehole."@en ;
+    skos:member bhse:hydraulic-fracturing-on-notice;
+    skos:prefLabel "PGGD-03"@en .
+
+bhse:pggd-04 a skos:Collection ;
+    skos:definition "The borehole events that may be reported via a PGGD-02 form to change the status of a borehole."@en ;
+    skos:member bhse:hydraulic-fracturing-abandoned,

```

Comment:
@BlakePaul @LizDerrington 
The other possible states for the current PGGD-04 form are Partial Completion, Alteration and Other?
I think we need to add Partial Completion and Alteration.
Can we get rid of Other? Or replace it with something more meaningful?

Reply:
partial and altered added
other added as Hydraulic fracturing related events